main/ZgatewayBT.ino

```diff
@@ -1048,6 +1048,7 @@ JsonObject& process_bledata(JsonObject& BLEdata) {
     Log.trace(F("Checking BLE service data validity" CR));
     const char* service_data = (const char*)(BLEdata["servicedata"] | "");
     if (valid_service_data(service_data)) {
+      int service_len = strlen(service_data);
```

Comment:
Could you use service_len to replace the others strlen(service_data)?

Reply:
Of course, originally I planned to do it in a separate pull request.
I will update the branch late afternoon.

Reply:
>  I planned to do it in a separate pull request.

That would be perfect

Reply:
Ok, do you plan to merge the branch today?
I will create the new branch and pull request after the merge.